### PR TITLE
fix #4163: feat(visualization) Always show default metrics in visualizations

### LINKED ISSUE #4163: Always show default metrics in visualization
There are some metrics that are always computed by default and don't need to be chosen as primary or secondary. These are active hours, ad clicks, etc.

We should always include these as "Secondary Metrics"

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-690)
┆Issue Number: EXP-690

### PR DESCRIPTION
**Pull request by emtwo**

### CODE REVIEW

**Comment by emtwo** [comment]
Screen Shot 2020-12-09 at 9 27 29 AM
Screen Shot 2020-12-09 at 9 27 50 AM

File: app/experimenter/visualization/api/v3/views.py [hunk 1]
```diff
@@ -178,7 +188,12 @@ def generate_results_object(data, experiment):
             results[branch][BRANCH_DATA][metric][comparison].update(
                 {"lower": lower, "upper": upper, "point": point}
             )
-    return results, primary_metrics_set
+
+            if metric not in result_metrics:
+                metric_title = " ".join([word.title() for word in metric.split("_")])
```
Comment (emtwo) [inline app/experimenter/visualization/api/v3/views.py:193]: Note that this is temporary until we incorporate the metadata that Jetstream generates for metrics

**Review by jaredlockhart — APPROVED** [review]
lgtm 👍

**Comment by jaredlockhart** [comment]
Also thank you for putting screenshots in prs!  Makes it so much easier to reason about 💯

File: app/experimenter/nimbus-ui/src/components/PageResults/index.tsx [hunk 2]
```diff
@@ -89,7 +89,17 @@ const AnalysisAvailable = ({
           <TableMetricSecondary
             key={probeSet?.slug}
```
Comment (LZoog) [inline app/experimenter/nimbus-ui/src/components/PageResults/index.tsx:90]: We can probably change `probeSet?.slug` to `probeSet!.slug` for consistency with L92

File: app/experimenter/nimbus-ui/src/components/TableMetricSecondary/index.test.tsx [hunk 3]
```diff
@@ -53,7 +57,9 @@ describe("TableMetricSecondary", () => {
       <RouterSlugProvider mocks={[mock]}>
         <TableMetricSecondary
           results={mockAnalysis().overall}
-          probeSet={experiment.primaryProbeSets![0]!}
+          probeSetSlug={experiment!.secondaryProbeSets![0]!.slug}
```
Comment (LZoog) [inline app/experimenter/nimbus-ui/src/components/TableMetricSecondary/index.test.tsx:60]: Jody updated the experiment mock where we no longer have to tell TS it's not falsey. You should be able to update all instances of `experiment!` to just `experiment` in your tests/stories.

**Review by LZoog — COMMENTED** [review]
LGTM, would like the `experiment!` update but feel free to merge at will after that with Jared's r+ :)

**Pull request merged**